widgetssdk/src/main/java/com/glia/widgets/call/CallView.java

```diff
@@ -780,10 +796,6 @@ private void showEndEngagementDialog(String operatorName) {
                     if (callController != null) {
                         callController.endEngagementDialogYesClicked();
                     }
-                    if (onEndListener != null) {
```

Comment:
Survey is optional? What will happen if engagement should end without survey

Reply:
It handles in `CallController`.
Anyway, `onEndListener.onEnd()` calls from the `destroyView()`.